supabase/migrations/20260816120150_org_member_read_billing_default.sql

```
@@ -0,0 +1,499 @@
+SELECT org_reach.user_id, org_reach.org_id
+FROM org_reach
+LEFT JOIN users_with_billing uwb
+  ON uwb.user_id = org_reach.user_id
+ AND uwb.org_id = org_reach.org_id
+WHERE uwb.user_id IS NULL;
+
+CREATE INDEX ON public._tmp_needs_org_read_billing (user_id, org_id);
+
+-- 3a) Extend existing org-scoped roles (one binding per principal/org).
+INSERT INTO public.role_permissions (role_id, permission_id)
+SELECT DISTINCT src.role_id, p.id
```

SQLFluff (4.2.2)

[error] Select targets should be on a new line unless there is only one select target.

(LT09)